packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -368,7 +368,10 @@ const CFDPasswordForm = observer(
         const { isDesktop } = useDevice();
         const { jurisdiction_selected_shortcode } = useCfdStore();
         const [checked, setChecked] = React.useState(false);
-        const need_tnc = jurisdiction_selected_shortcode !== 'svg' && account_type.category === CATEGORY.REAL;
+        const need_tnc =
+            jurisdiction_selected_shortcode !== 'svg' &&
+            account_type.category === CATEGORY.REAL &&
+            platform !== CFD_PLATFORMS.DXTRADE;
```

Comment:
```suggestion
            platform === CFD_PLATFORMS.MT5;
```

looks like targeting mt5 rather than excluding dxtrade

Reply:
Okieee 👌 so only mt5 accounts need to accept the TNC?

Reply:
yup!